linux/utils/enable_auto_login.yml

```diff
@@ -55,7 +55,7 @@
   ansible.builtin.set_fact:
     dm_conf_path: "/etc/lightdm/lightdm.conf"
     dm_conf_section: "Seat:*"
-    dm_autologin_options: {"autologin-user": "{{ autologin_user }}"}
+    dm_autologin_options: {"pam-autologin-service": "lightdm-autologin", "user-session": "default", "autologin-user": "{{ autologin_user }}"}
```

Comment:
Only keep `autologin-user` should work. Please remove "pam-autologin-service" and "user-session", then retry